app/views/saml_idp/idp/saml_post.html.erb

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset='utf-8'>
     <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
+    <title><%= t("layouts.application.header.title") %></title>
```

Comment:
Ditto portal login

Reply:
> Ditto portal login

Great. I decided to use the service name in these instances as we weren't yielding any page title values to the layout page.